### ISSUE TITLE
Limit size of input in error messages

### ISSUE THREAD

**Issue by aoudiamoncef** [massalabs/massa#2798]
We should define a limit for input payload in error messages.
```
2022-07-11T09:29:17.010015Z  INFO massa_node: Node version : TEST.12.0
2022-07-11T09:29:17.044796Z  INFO massa_bootstrap::client: Start bootstrapping from 198.27.74.52:31245
2022-07-11T09:29:17.212989Z  WARN massa_bootstrap::client: Error while bootstrapping: general bootstrap error: Parsing Error: Failed BootstrapServerMessage deserialization / Failed u32 deserialization / Fail / Input: [113, 89, 111, 117, 114, 32, 108, 97, 115, 116, 32, 98, 111, 111, 116, 115, 116, 114, 97, 112, 32, 111, 110, 32, 116, 104, 105, 115, 32, 115, 101, 114, 118, 101, 114, 32, 119, 97, 115, 32, 50, 50, 48, 54, 46, 55, 48, 52, 54, 57, 48, 56, 54, 51, 115, 32, 97, 103, 111, 32, 97, 110, 100, 32, 121, 111, 117, 32, 104, 97, 118, 101, 32, 116, 111, 32, 119, 97, 105, 116, 32, 52, 48, 57, 57, 51, 46, 50, 57, 53, 51, 48, 56, 57, 55, 56, 115, 32, 98, 101, 102, 111, 114, 101, 32, 114, 101, 116, 114, 121, 105, 110, 103, 46]
```

I don't know if it's interesting to give back a vector of byte to the user ? may be we don't have to return the input ?

**Comment by AurelienFT** [comment]
Will be fixed in #2772

**Comment by damip** [comment]
done
